### PR TITLE
README.md - add teamcamp in list

### PR DESCRIPTION
**Pull request by aniket-pixer**
## Requirements

 * [ ] This is Software as a Service not self hosted
 * [ ] It has a free tier not just a free trial
 * [ ] Pricing information is clearly visible without signup or phone calls
 * [ ] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,6 +451,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Tefter](https://tefter.io) - Bookmarking app with a powerful Slack integration. Free for open-source teams.
   * [TeleType](https://teletype.oorja.io/) — share terminals, voice, code, whiteboard, and more. no sign-in is required for end-to-end encrypted collaboration for developers.
   * [TimeCamp](https://www.timecamp.com/) - Free time tracking software for unlimited users. Easily integrates with PM tools like Jira, Trello, Asana, etc.
+  * [Teamcamp](https://www.teamcamp.app) - All-in-one project management application for software development companies.
```
Comment (ripienaar) [inline README.md:454]: All I see is a free trial, mind sharing a link to the free tier?

**Comment by ripienaar** [comment]
@aniket-pixer i already left you comment.

**Comment by aniket-pixer** [comment]
@ripienaar https://www.teamcamp.app/pricing  We haven't mentioned free usage before; we have just updated with a better design.

**Comment by aniket-pixer** [comment]
@ripienaar  Created another PR : https://github.com/ripienaar/free-for-dev/pull/3520

**Pull request closed without merging**